rust/kcl-lib/src/std/transform.rs

```diff
@@ -28,9 +30,9 @@ pub async fn scale(exec_state: &mut ExecState, args: Args) -> Result<KclValue, K
         ]),
         exec_state,
     )?;
-    let scale_x = args.get_kw_arg_opt("x")?;
-    let scale_y = args.get_kw_arg_opt("y")?;
-    let scale_z = args.get_kw_arg_opt("z")?;
+    let scale_x: Option<TyF64> = args.get_kw_arg_opt_typed("x", &RuntimeType::length(), exec_state)?;
+    let scale_y: Option<TyF64> = args.get_kw_arg_opt_typed("y", &RuntimeType::length(), exec_state)?;
+    let scale_z: Option<TyF64> = args.get_kw_arg_opt_typed("z", &RuntimeType::length(), exec_state)?;
```

Comment:
Should this be count? I don't think it makes sense to scale by a length because the result would be length squared.

Reply:
I think in practice points are always lengths and we don't differentiate between points and vectors :-( It could probably be Any though